src/js/player.js

```diff
@@ -2415,7 +2380,7 @@ class Player extends Component {
    *         The source MIME type
    */
   currentType() {
-    return this.currentType_ || '';
+    return this.currentSource() && this.currentSource().type || '';
```

Comment:
using `ack` and `ripgrep` returns no results for `this.currentType_` now, so I don't think so.